### PR TITLE
Regex performance improvements

### PR DESCRIPTION
**Pull request by Ladsgroup**
On nl, fr, de, it, pt, es, uk

### CODE REVIEW

File: revscoring/languages/portuguese.py [hunk 1]
```diff
@@ -108,7 +108,7 @@
     r"adoro",  # love
     r"aki",  # here
     r"amo",  # master
-    r"(b+l+a+h*)+",  # bla, blah, bbblllaaaahhhhhblah
+    r"(?:b+?l+?a+?h*)+",  # bla, blah, bbblllaaaahhhhhblah
```
Comment (he7d3r) [inline revscoring/languages/portuguese.py]: Why not to do the same for the last plus sign?

File: revscoring/languages/portuguese.py [hunk 2]
```diff
@@ -120,9 +120,9 @@
     r"fixe",  # cool
     r"gajo",  # dude
     r"h[aiou](h[aeiou])*", r"h[e](h[aeiou])+",  # hi, ha, hehe, hohoho
-    r"k+",  # k, kkkkkkkkkkkkkkk
+    r"k+?",  # k, kkkkkkkkkkkkkkk
     r"lindo",  # pretty
-    r"l+([uo]+l+)+",  # lol, LOLOLOL, LLLLoOOoLLL
+    r"l+?([uo]+?l+?)+",  # lol, LOLOLOL, LLLLoOOoLLL
```
Comment (he7d3r) [inline revscoring/languages/portuguese.py]: we can probably change this (and a few others) to `(?:` too

File: revscoring/languages/spanish.py [hunk 3]
```diff
@@ -176,7 +176,7 @@
     r"polvo",  # nut (testical)
     r"poto",  # ass
     r"prostituta",  # prostitute
-    r"\w*put([ao]+|isim[oa]|iza)(s|n)?\w*",  # bitch/whore
+    r"put([ao]+|isim[oa]|iza)(s|n)?\w*",  # bitch/whore
```
Comment (he7d3r) [inline revscoring/languages/spanish.py]: Isn't better to change `(s|n)` to `[sn]`?

File: revscoring/languages/portuguese.py (hunk 2 above)
Comment (Ladsgroup) [inline revscoring/languages/portuguese.py]: It's not a huge improvement. Around 1% overall.

**Pull request merged**